### PR TITLE
230 arima failing

### PR DESCRIPTION
**Pull request by IanGrimstead**
Added test which blows ARIMA without fix...

### CODE REVIEW

**Comment by IanGrimstead** [comment]
Argh ARIMA test which passes locally fails on remote...

File: config/stopwords_n.txt [hunk 1]
```diff
@@ -2,4 +2,5 @@ situation
 consist
 first
 plurality
-second
\ No newline at end of file
+second
+top
```
Comment (thanasions) [inline config/stopwords_n.txt]: this one (top) is naughty, causing the tests to fail! So delete.

Comment (IanGrimstead) [reply config/stopwords_n.txt]: Wi-erd must have pulled from the wrong branch!

**Comment by IanGrimstead** [comment]
Urgh. ARIMA test failing on build but passing locally... work in progress...

**Comment by IanGrimstead** [comment]
Urgh local tests pass, remote travis builds fail - different results from ARIMA :(

**Pull request merged**